### ISSUE TITLE
Iron trapdoor render error

### ISSUE THREAD

**Issue by NJDaeger** [llbit/chunky#410]
When iron trapdoors are on the top half like in some of the windows on this image:
http://imgur.com/gallery/IFyohVR
There is a bug when one face of the texture acts as if it's on the bottom half of the block. In simpler terms, the block is on the top half and one face if appearing to be on the bottom half. 

Note: I am unsure whether this happens with wood trapdoors or not, and I don't know if it does it for any other position either. I will do more testing if needed.

**Comment by llbit** [comment]
I see what you mean:

![screenshot 2016-12-27 13 37 02](https://cloud.githubusercontent.com/assets/2757386/21499830/91b60132-cc39-11e6-963a-69b4238575ef.png)
